### PR TITLE
add fragment cache

### PR DESCRIPTION
**Pull request by ryota-sasabe**
Connect to https://github.com/bm-sms/daimon-news-misc/issues/180

### CODE REVIEW

File: app/views/posts/show.html.haml [hunk 1]
```diff
@@ -10,33 +10,34 @@
...
-    %p.thumbnail= image_tag @post.thumbnail_url
-  .post__body
-    = render_markdown(@pages.map(&:body).join)
-
-  - if @pages.total_pages > 1
-    %nav.pagination
-      = paginate @pages, theme: :post
-
-  = render 'application/banner', position: :press_article_bottom
+- cache [@post, @pages.current_page], expires_in: 1.minutes do
```
Comment (mtsmfm) [inline app/views/posts/show.html.haml:13]: This cache depends on credits I think

File: config/environments/development.rb [hunk 2]
```diff
@@ -38,4 +38,6 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  config.action_controller.perform_caching = true
```
Comment (mtsmfm) [inline config/environments/development.rb]: Don't override config :trollface: 
Update L14

**Comment by mtsmfm** [comment]
Almost great ⭐️ 
Where will you store the cache?

File: app/views/posts/show.html.haml (hunk 1 above)
Comment (ryota-sasabe) [reply app/views/posts/show.html.haml:13]: Since the cache expires in 1 minute, it is not strictly considered.

**Comment by ryota-sasabe** [comment]
Save cache to Redis.

**Pull request merged**